src/main/java/com/massivecraft/factions/cmd/CmdShowClaims.java

```diff
@@ -0,0 +1,73 @@
+package com.massivecraft.factions.cmd;
+
+import com.massivecraft.factions.Board;
+import com.massivecraft.factions.FLocation;
+import com.massivecraft.factions.perms.PermissibleAction;
+import com.massivecraft.factions.struct.Permission;
+import com.massivecraft.factions.tag.Tag;
+import com.massivecraft.factions.util.TL;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CmdShowClaims extends FCommand {
+
+    private static Pattern WORLD_PATTERN = Pattern.compile("\\{showclaims_(.*)}");
+
+    public CmdShowClaims() {
+        this.aliases.add("showclaims");
+
+        this.requirements = new CommandRequirements.Builder(Permission.SHOWCLAIMS)
+                .memberOnly()
+                .withAction(PermissibleAction.TERRITORY)
+                .noDisableOnLock()
```

Comment:
Should we disable on lock?

Reply:
This is just listing, so it does not need to disable on lock.